code/pages/author.fs

```diff
@@ -0,0 +1,27 @@
+module FsSnip.Pages.Author
+
+open Suave
+open System
+open System.Web
+open FsSnip.Utils
+open FsSnip.Data
+
+// -------------------------------------------------------------------------------------------------
+// Author page - domain model
+// -------------------------------------------------------------------------------------------------
+
+type AuthorModel =
+  { Author : string
+    Snippets : seq<Snippet> }
+
+// -------------------------------------------------------------------------------------------------
+// Loading author page information (snippets by the given author)
+// -------------------------------------------------------------------------------------------------
+
+let showSnippets (author) =
+  let a = System.Web.HttpUtility.UrlDecode author
+  publicSnippets
+  |> Seq.filter (fun s -> s.Author.Equals(a, StringComparison.InvariantCultureIgnoreCase))
+  |> (fun s ->
```

Comment:
My personal preference is to avoid pipelines if I have to write `|> (fun x -> ...)` and instead write multiple let bindings with variables. But I know a couple of people who like piping into fun :-) so I'm happy to merge this no matter which style it uses.